client/src/app/page/create-update-user-page/create-update-side-bar-info/dropdown-info/dropdown-info.component.ts

```diff
@@ -11,9 +12,36 @@ export class DropdownInfoComponent implements OnInit {
   @Output() readonly selected = new EventEmitter<any>();
   @Input() required: boolean;
   title: string;
+  @Input() positionIdentifier: string;
+  @Input() update: string;
+  @Input() roles: boolean;
+
+  constructor(private userService: UserService) {
+  }
 
   ngOnInit(): void {
-    this.title = 'Choose';
+    if (!this.update) {
+      this.title = 'Choose';
+    }
+    if (this.positionIdentifier === 'position') {
+      this.userService.chosenDepartment.subscribe(
+        () => this.title = 'Choose'
+      );
+    }
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.pairList && changes.pairList.currentValue && changes.pairList.currentValue.length > 0) {
```

Comment:
`changes.pairList && changes.pairList.currentValue && changes.pairList.currentValue.length > 0)` move it to const